cluster/manager_delete.go

```diff
@@ -352,6 +364,11 @@ func (m *Manager) deleteCluster(ctx context.Context, cluster CommonCluster, forc
 	// delete from proxy from kubeProxyCache if any
 	m.DeleteKubeProxy(cluster)
 
+	err = deleteUnusedSecrets(cluster)
```

Comment:
This supersedes `ClusterModel.preDelete` thus `ClusterModel.preDelete` can be deleted.

Reply:
I think I have already deleted it in [this](https://github.com/banzaicloud/pipeline/pull/1638/commits/cb32a0b6f7feb5d35cba22d9e31e216312083adb) commit.